dist/css/bootstrap.css

```diff
@@ -1647,6 +1494,13 @@ pre code {
 .table-dark.table-hover tbody tr:hover {
   background-color: rgba(255, 255, 255, 0.075);
 }
+.table-responsive {
```

Comment:
@matthew-dean Any idea why this is being moved?  It's happening in mine as well.  It's generated by the `each()` at https://github.com/seanCodes/bootstrap-less-port/pull/12/files#diff-c4fb9381184254ae43a8b5d8ea35a322R194.

Reply:
That's really peculiar. It must be some side effect of the `each()` implementation. Although looking at the function, nothing stands out. It creates an array of generated rulesets. So there must be a bug somewhere in Less around where rulesets are flattened?

Reply:
Never mind, this has to be a bug in one of the `@plugin` functions. Because if you just turn this into straight mixins, you get expected output:

See: https://gist.github.com/matthew-dean/9add256671e64b2c1d285129adb33658

Reply:
Ah, yeah, actually this being up here is more expected I think.  The `.table-responsive > .table-bordered {...}` block below (see link) should probably be up at the top as well.  @seanCodes: worth noting.

https://github.com/seanCodes/bootstrap-less-port/pull/12/files/8337dccea789ed6d6d2b8ff5b45737a3ac60877d#diff-6972cbb37f0ec48771217d4915ea7e6fR1552

Reply:
So it seems Bootstrap's intent with that loop is explicitly to have each iteration of the loop generate the _next_ breakpoint's rules, with the final breakpoint instead generating un-infixed rules.  Digging further, it looks to be a bug in Less (see https://github.com/less/less.js/issues/3340).